polaris-react/src/components/Text/README.md

```diff
@@ -268,5 +268,145 @@ Use to de-emphasize a piece of text that is less important to merchants than oth
   No supplier listed
```

Comment:
If we want this to show up on the website? I would recommend moving this content to `polaris.shopify.com/content/components/text.md`

Reply:
So what content should go in this `README`?

Reply:
The examples are the final part to migrate.

Reply:
Ok got it! Thanks! 🧀